<launch>

    <!-- ORB-SLAM3 -->
    <node name="orb_slam3_mono_inertial" pkg="orb_slam3_ros_wrapper" type="orb_slam3_ros_wrapper_mono_inertial" output="screen">
        <!-- for EuRoC dataset -->
      <remap from="/camera/image_raw"         to="/uav1/tellopy_wrapper/rgb/image_raw" />
      <remap from="/imu"                      to="/uav1/hw_api/imu"/>

        <!-- Parameters for original ORB-SLAM3 -->
        <param name="voc_file"      type="string"   value="$(find orb_slam3_ros_wrapper)/config/ORBvoc.txt" />
        <param name="settings_file" type="string"   value="$(find orb_slam3_ros_wrapper)/config/EuRoC_mono_inertial_rectified.yaml" />
        <param name="do_equalize"   type="bool"     value="false" />

        <!-- Parameters for ROS -->
        <param name="map_frame_id"  type="string"   value="world" />
        <param name="pose_frame_id" type="string"   value="uav1/fcu" />
    </node>

</launch>
